_includes/header.html

```diff
@@ -1,71 +1,118 @@
 <div class="usa-header usa-header--extended bg-white">
   <div class="usa-navbar container">
     <div class="usa-logo" id="extended-logo">
-      <a href="{{ '/' | prepend: site.baseurl }}" title="Home" aria-label="Home">
-        <img src="{{ site.baseurl }}/assets/img/logo.svg" class="usa-logo__img" alt="login.gov">
+      <a
+        href="{{ '/' | prepend: site.baseurl }}"
+        title="Home"
+        aria-label="Home"
+      >
+        <img
+          src="{{ site.baseurl }}/assets/img/logo.svg"
+          class="usa-logo__img"
+          alt="login.gov"
+        />
       </a>
     </div>
 
     <button class="usa-menu-btn">Menu</button>
   </div>
 
-  <nav role="navigation" class="usa-nav" >
-    <div class="usa-nav__inner container">
-      <button class="usa-nav__close"><img src="{{ site.baseurl }}/assets/img/close.svg" alt="Close"></button>
+  <nav role="navigation" class="usa-nav">
+    <div class="usa-nav__containter container">
```

Comment:
Am I correct in thinking this should be `usa-nav-container`?

```suggestion
    <div class="usa-nav-container">
```

(Also noting there are some [related styles introduced here](https://github.com/18F/identity-site/pull/454/files#diff-0a93bce0c8bdd3bb271e40b6647c33a25eef140ac03315234944b889b15b21d7R28) which may be impacted by this change)

Is `container` a BassCSS class? I'm starting to think my previous script pattern may have been missing a few 😅

Reply:
`container` is a BassCSS class, however I kept it because it's used everywhere! I applied the new grid to the `container` class in layout.scss. [See line 7](https://github.com/18F/identity-site/pull/454/files#diff-0a93bce0c8bdd3bb271e40b6647c33a25eef140ac03315234944b889b15b21d7R7).

Reply:
> Am I correct in thinking this should be `usa-nav-container`?
Actually, no. USWDS uses the BEM naming methodology. In this case, `usa-nav` is the block, `__container` is the element. `-container` would be a modifier.

Reply:
Correct, I renamed it because `usa-nav-container` has it's own padding. Instead of overriding that styling, I created a different container name (using BEM).